pytorch_toolkit/object_detection/requirements.txt

```diff
@@ -1,12 +1,13 @@
 cython==0.29.12
-numpy==1.16.4
-torch==1.1
-mmcv==0.2.10
-tensorboardX==1.8
-tensorboard==1.14.0
+matplotlib==3.1.1
+mmcv==0.2.16
+networkx==2.3
+numpy==1.17.5
 opencv-python==4.1.0.25
+ptflops==0.3
```

Comment:
Is `ptflops` really required? Isn't mmdetection's counter used for these models?